<launch>
  <node pkg="move_arm" type="move_arm_action" output="screen">    

    <remap from="robot_description" to="robot_description" />
    <remap from="collision_map" to="collision_map_occ" />
    <remap from="collision_map_update" to="collision_map_occ_update" />

    <remap from="arm_ik" to="pr2_ik_right_arm/ik_service" />	

    <remap from="get_valid_state" to="find_valid_state" />
    
    <remap from="controller_start"  to="/joint_waypoint_controller/TrajectoryStart" />
    <remap from="controller_query"  to="/joint_waypoint_controller/TrajectoryQuery" />
    <remap from="controller_cancel" to="/joint_waypoint_controller/TrajectoryCancel" />
    
    <remap from="get_motion_plan" to="plan_kinematic_path" />
    
    <param name="arm" type="string" value="right_arm" />
    <param name="show_collisions" type="bool" value="true" />

    <param name="refresh_interval_collision_map" type="double" value="20.0" />
    <param name="refresh_interval_kinematic_state" type="double" value="1.0" />
    <param name="bounding_planes" type="string" value="0 0 1 -0.01" />
  </node>

  <include file="$(find move_arm)/launch/gripper_rarm.launch" />
  
</launch>
